db/seeds.rb

```diff
@@ -2,6 +2,4 @@
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
-#
-admin = AdminUser.create!(firstname: "Muhammad", lastname: "Haroon", email: "admin@fintech.com", password: "Password123!@#")
-
+admin= User.where(email: "admin@fintech.com").first_or_create(firstname: "Muhammad", lastname: "Haroon", password: "Password123!@#", role: "admin")
```

Comment:
space missing before `=`
Make sure the line does not go beyond 114 columns. In such cases manage it like this.

User
  .where(email: "admin@fintech.com")
  .first_or_create(firstname: "Muhammad", lastname: "Haroon", password: "Password123!@#", role: "admin")

Also instead of the `admin` string use a constant I explained above like User::ROLES[:admin] So if we change it in future. it gets changed at all places easily.